src/scene/container/container-mixins/cacheAsTextureMixin.ts

```diff
@@ -79,4 +76,22 @@ export const cacheAsTextureMixin: Partial<Container> = {
     {
         this.renderGroup?.updateCacheTexture();
     },
+
+    /**
+     * Allows backwards compatibility with pixi.js below version v8. Use `cacheAsTexture` instead.
+     * @deprecated
+     */
+    get cacheAsBitmap(): boolean
+    {
+        return this.isCachedAsTexture;
+    },
+
+    /**
+     * @deprecated
+     */
+    set cacheAsBitmap(val: boolean)
+    {
+        warn('cacheAsBitmap is deprecated, use cacheAsTexture instead.');
```

Comment:
Can we use deprecation instead?
```suggestion
        // #if _DEBUG
        deprecation('cacheAsBitmap is deprecated, use cacheAsTexture instead.');
        // #endif
```

Reply:
nice!!